frontend/src/queries/nodes/InsightViz/EditorFilters.scss

```diff
@@ -9,14 +9,17 @@
     padding: 1rem;
     margin-bottom: 1rem;
     background: var(--bg-light);
+    container-type: inline-size;
 
     .EditorFilters {
         > * + * {
             margin-top: 1rem;
         }
-        @include screen($md) {
+
+        @container (min-width: 768px) {
```

Comment:
how is support for `@container` vs our visitors actual browsers?

Nothing in IE11 from the look of it... but we maybe don't care about that?

Ah

```
"browserslist": {
        "development": [
            "last 2 chrome versions",
            "last 2 firefox versions",
            "last 2 edge versions"
        ],
        "production": [
            "defaults"
        ]
    },
```

In the package.json (I wonder if we lint based on this?)

So we should be OK